.gitignore

```diff
@@ -3,4 +3,6 @@ build
 out
 .idea
 *.iml
-*~
\ No newline at end of file
+*~
+# sometimes antlr barfs, and I don't want to add those files by accident
+src/main/gen/
```

Comment:
in general a good hygiene is to just add `gen`

Reply:
Are you saying my .gitignore should have just `gen/`?